capa/render/vverbose.py

```diff
@@ -56,66 +57,66 @@ def render_locations(ostream, layout: rd.Layout, locations: Iterable[frz.Address
     if len(locations) == 0:
         return
 
-    ostream.write(" @ ")
+    console.print(" @ ", end="")
     location0 = locations[0]
 
     if len(locations) == 1:
         location = locations[0]
 
         if location.type == frz.AddressType.CALL:
             assert isinstance(layout, rd.DynamicLayout)
-            ostream.write(hanging_indent(v.render_call(layout, location), indent + 1))
+            console.print(hanging_indent(v.render_call(layout, location), indent + 1), end="")
```

Comment:
kinda sucks that we have to add all these `, end=""` args, but I understand why. Do you think it would be worth subclassing Console to provide a `.write()` so that the intent is more clear?

Reply:
(I feel like the trailing `end=` kwarg is rather hidden and hard to notice visually)

Reply:
I share the same concern here. The `, end=""` statements are scattered around and it is not easy to know which is a newline and what isn't.

I'd consider subclassing `rich.console.Console` and introducing `write` and `writeln` methods in `capa/render/utils`.